samples/nrf9160/aws_fota/README.rst

```diff
@@ -88,12 +88,12 @@ This application, which is part of `nRF Connect for Desktop`_, provides a certif
 
 Use LTE Link Monitor to write the certificates to the kit:
 
-1. Make sure that you have the :ref:`at_host_sample` sample programmed on your kit.
-#. Put the modem in offline state.
-#. Paste the three certificates into the respective fields.
-#. Choose a security tag.
-#. Click :guilabel:`Update certificates`.
-#. Before programming the sample, make sure to configure the :option:`security tag <CONFIG_CERT_SEC_TAG>` to the one that you chose.
+   1. Make sure that you have the AT client sample programmed on your kit.
+   #. Put the modem in offline state.
+   #. Paste the three certificates into the respective fields.
+   #. Choose a security tag.
+   #. Click :guilabel:`Update certificates`.
+   #. Before programming the sample, make sure to configure the :option:`security tag <CLOUD_CERT_SEC_TAG>` to the one that you chose.
```

Comment:
should this be changed?

Reply:
could very well be.
There were some minor conflicts on the release rst, and this rst file, so I chose what appeared to be the best option.

But I have no idea what should gi here instead. @umapraseeda or @bjda suggestions ?